ipetcare.spa/src/features/auth/containers/loginForm.tsx

```diff
@@ -23,17 +24,12 @@ export function LoginForm() {
     e.preventDefault()
     setError(false)
     try {
-      const response = await axios.post(BASE_URL + '/users/login', {
-        email: login,
-        password,
-      })
+      const response = await Users.login({ email: login, password })
 
-      const user: UserState = response.data as UserState
-
-      dispatch(setUser(user))
-      saveUserState(user)
-      setTokenInHeader(user.token)
-      switch (user.role.toLowerCase()) {
+      dispatch(setUser(response))
+      saveUserState(response)
+      setTokenInHeader(response.token)
+      switch (response.role.toLowerCase()) {
         case 'owner':
```

Comment:
W sumie można by to do enuma wyjąć, na pewno będziemy w wielu miejscach na froncie sprawdzać role name

Reply:
Tfu enumy XD jak coś to do type